dotnet/MinimalWeather/Program.cs

```diff
@@ -13,8 +13,10 @@
 var app = builder.Build();
 app.UseCors();
 
-using var httpClient = new HttpClient();
-httpClient.BaseAddress = new Uri("https://atlas.microsoft.com/weather/");
+using var httpClient = new HttpClient
+{
+    BaseAddress = new("https://atlas.microsoft.com/weather/")
```

Comment:
I used the object initialize in the initial versions, but it adds two lines and adds funky syntax (if you're not used to it).